test/IDE/complete_from_swift_module.swift

```diff
@@ -99,3 +105,20 @@ func testPostfixOperator1(x: Int) {
 // TOP_LEVEL_1-DAG: Decl[GlobalVar]/Local:     hiddenImport[#Int#]{{; name=.+$}}
 // TOP_LEVEL_1-DAG: Decl[GlobalVar]/OtherModule[foo_swift_module]:     globalVar[#Int#]{{; name=.+$}}
 // TOP_LEVEL_1: End completions
+
+struct Foo: Swift.Array.#^STDLIB_TYPE_QUALIFIED^# {}
+// STDLIB_TYPE_QUALIFIED: Begin completions
+// STDLIB_TYPE_QUALIFIED: Decl[TypeAlias]/CurrNominal: Index[#Int#]; name=Index
+// STDLIB_TYPE_QUALIFIED: Decl[TypeAlias]/CurrNominal: Element[#Element#]; name=Element
+// STDLIB_TYPE_QUALIFIED: Keyword/None: Type[#Array.Type#]; name=Type
+// STDLIB_TYPE_QUALIFIED: End completions
```

Comment:
I believe this test case works even before this change.
Could you add a test for `Swift.#^TOKEN^#` at type position as well?

Reply:
Yeah, it was already working. Added the extra test.